src/smartcontracts/interaction.local.net.spec.ts

```diff
@@ -184,6 +185,118 @@ describe("test smart contract interactor", function () {
         assert.isTrue(typedBundle.returnCode.equals(ReturnCode.Ok));
     });
 
+    it("should interact with 'basic-features' (local testnet) using the SmartContractTransactionsFactory", async function () {
```

Comment:
Test is very nice. For education purposes, maybe it's better to have it using the new transactions factory.